spec/spec.md

```diff
@@ -457,11 +457,43 @@ The following properties are for use at the top-level of the resource — all
 other properties that are not defined below MUST be ignored:
 
 - `name` - The resource ****MAY**** contain this property, and if present its
-value ****SHOULD**** be a human-friendly name that describes what the
-Presentation Definition pertains to.
+  value ****SHOULD**** be a human-friendly name that describes what the
+  Presentation Definition pertains to.
 - `purpose` - The resource ****MAY**** contain this property, and if present its
-value ****MUST**** be a string that describes the purpose for which the
-Presentation Definition's inputs are being requested.
+  value ****MUST**** be a string that describes the purpose for which the
+  Presentation Definition's inputs are being requested.
+- The resource ****MAY**** include a `format` property, and its value 
+  ****MUST**** be an object with one or more properties matching the registered 
+  [Credential Format Designations](#credential-format-designations) to inform the 
+  Holder of the credential format configurations the Verifier can process. The 
+  value for each property included ****MUST**** be an object composed as follows:
+    - The object ****MAY**** include an `signature` property, and its value 
```

Comment:
I know it is a minor quibble, but I dislike `signature` as the name of this property, I expect its value to be a signature, not a list of supported submission signature schemes.
However, I admit that `supported_submission_signature_scheme` is probably too verbose.

What about `signature_type`?

Reply:
Algorithms?

Reply:
+1 to `signature_type`